src/color/base.json

```diff
@@ -2,150 +2,346 @@
+          "version": "3.1.0",
+          "reference": "color-brand-neutral-400",
+          "comment": "{comments.reference.comment} color-brand-neutral-400"
         },
-        "200": {
-          "value": "dbdbdb",
+        "500": {
+          "value": "#626b79",
           "public": false,
-          "neutral": true,
-          "usage": "Neutral accent",
-          "wcag": "n/a",
           "deprecated": true,
           "version": "3.1.0",
-          "reference": "color-brand-gray-200",
-          "comment": "{comments.reference.comment} color-brand-gray-200"
-        },
-        "300": {
-          "value": "b2b2b2",
+          "reference": "color-brand-neutral-500",
+          "comment": "{comments.reference.comment} color-brand-neutral-500"
+        }
+      },
+      "grey": {
+        "100": {
+          "value": "#f8f8f8",
+          "version": "3.1.0",
           "public": false,
           "neutral": true,
           "usage": "Neutral accent",
           "wcag": "n/a",
           "deprecated": true,
-          "version": "3.1.0",
-          "reference": "color-brand-gray-300",
-          "comment": "{comments.reference.comment} color-brand-gray-300"
+          "reference": "color-brand-grey-100",
+          "comment": "{comments.reference.comment} color-brand-grey-100"
         },
-        "400": {
-          "value": "767676",
+        "200": {
+          "value": "#dbdbdb",
           "public": false,
           "neutral": true,
           "usage": "Neutral accent",
           "wcag": "n/a",
           "deprecated": true,
           "version": "3.1.0",
-          "reference": "color-brand-gray-400",
-          "comment": "{comments.reference.comment} color-brand-gray-400"
+          "reference": "color-brand-grey-200",
+          "comment": "{comments.reference.comment} color-brand-grey-200"
         },
         "500": {
-          "value": "222222",
+          "value": "#222222",
           "public": false,
           "neutral": true,
           "usage": "Neutral accent",
           "wcag": "n/a",
           "deprecated": true,
           "version": "3.1.0",
-          "reference": "color-brand-gray-500",
-          "comment": "{comments.reference.comment} color-brand-gray-500"
-        }
-      },
-      "neutral": {
-        "400": {
-          "value": "9fabbb",
-          "public": false,
-          "deprecated": true,
-          "version": "3.1.0",
-          "reference": "color-brand-neutral-400",
-          "comment": "{comments.reference.comment} color-brand-neutral-400"
-        },
-        "500": {
-          "value": "626b79",
-          "public": false,
-          "deprecated": true,
-          "version": "3.1.0",
-          "reference": "color-brand-neutral-500",
-          "comment": "{comments.reference.comment} color-brand-neutral-500"
+          "reference": "color-brand-grey-500",
+          "comment": "{comments.reference.comment} color-brand-grey-500"
         }
       }
     },
     "state": {
```

Comment:
According to figma, `state` tokens should be deprecated and there should be new `base` tokens